sites/all/themes/osto_web/templates/html.tpl.php

```diff
@@ -0,0 +1,19 @@
+<?php print $doctype; ?>
+<html lang="<?php print $language->language; ?>" dir="<?php print $language->dir; ?>"<?php print $rdf->version . $rdf->namespaces; ?>>
+<head<?php print $rdf->profile; ?>>
+  <script id="CookieConsent" src="https://policy.app.cookieinformation.com/uc.js" data-culture="DA" type="text/javascript"></script>
```

Comment:
Should this JS not have been added in `itk_siteimprove_init()` hook (which is activated on every page call) with add_js or in an `hook_preprocess` in `itk_siteimprove` so it is contained in the module. 

This is an hidden dependency.

Reply:
https://policy.app.cookieinformation.com/uc.js is not related in any way to siteimprove. It just adds cookie consent banner